subprojects/docs/src/docs/release/notes.md

```diff
@@ -36,9 +36,14 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
-<!--
-### Example breaking change
--->
+### Support for Maven optional dependencies
```

Comment:
❌ We should mention this in the new and noteworthy section as well, this is a much-requested improvement.

Reply:
I'm happy to read suggestions here, without duplicating what is in the potential breaking changes section.

Reply:
I'd just move most of the description up and just mention down here that "Honoring optional dependencies can change the resolution result of existing builds."

Reply:
done